tests-ci/tests/test_ws.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+let imports = {
+  scene: 'px:scene.1.js',
+  ws: 'ws',
+  assert: '../test-run/assert.js',
+  manual: '../test-run/tools_manualTests.js'
+};
+
+const importsPromise = px.import(imports);
+
+module.exports.beforeStart = () => importsPromise;
+
+importsPromise.then(im => {
+  imports = im;
+  imports.assert = imports.assert.assert;
+  if (imports && imports.manual.getManualTestValue() === true) {
+    imports.manual.runTestsManually(module.exports.tests, module.exports.beforeStart);
+  }
+});
+
+const child_url = `${px.getPackageBaseFilePath()}/helpers/test_childAppWs.js`;
+const echo_server_url = 'wss://echo.websocket.org';
```

Comment:
Does this site document that they might reject the connection if our CI is hitting it multiple times a day, every day?  Could we handle that error gracefully rather than failing, so as not to cause CI failures in that case?

Reply:
onError should fire to indicate if/when an issue occurs. Need to monitor CI results with this in mind, and change the test if/when we know what errors might be issued by the server.